### PR TITLE
PR#8: Finalising UI changes for the frontend

### LINKED ISSUE #30: Remove `inject` usage for stores

### PR DESCRIPTION
**Pull request by elit-altum**
Following features have been added:

1. Empty message if no runs, giving instructions on how to start the runs.
2. Color to the control buttons of runs
3. Close button to terminate the dashboard
4. All run events being wired to the run summary page showing cursor and item position
5. Logo for newman dashboard
6. Move away from `inject` and import the store locally

Bugs 🐛
1. Fix the horizontal scrollbars on the run-summary page

More work on ⭐ 
1. Add specific components for uniquely handling every run event

Closes #30,  #31, #39, #38

Run summary page 🌟:
image

### CODE REVIEW

File: dashboard/frontend/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    const handleCloseButton = () => {
+        if (confirm('Are you sure you want to exit?')) {
+            socket.emit("terminate");  
+        } 
+    };
+
+    return (
+        <nav className="flex items-center justify-between flex-wrap shadow px-4 py-4">
+            <Link href="/" passHref={true}>
+                <Image
```
Comment (coditva) [inline dashboard/frontend/components/Header.js]: This is failing the build for the frontend `npm run build:client`. 

```markdown
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Use any provider which supports Image Optimization (like Vercel).
    - Configure a third-party loader in `next.config.js`.
    - Use the `loader` prop for `next/image`.
  Read more: https://nextjs.org/docs/messages/export-image-api
    at /Users/coditva/Code/elit-altum/newman-dashboard/node_modules/next/dist/export/index.js:14:785
    at async Span.traceAsyncFn (/Users/coditva/Code/elit-altum/newman-dashboard/node_modules/next/dist/telemetry/trace/trace.js:6:584)
```

Ref: https://nextjs.org/docs/basic-features/image-optimization#loader

Comment (elit-altum) [reply dashboard/frontend/components/Header.js]: Fixed it!

File: dashboard/frontend/actions/index.js [hunk 2]
```diff
@@ -49,14 +57,26 @@ const mountSockets = (store) => {
     socket.on('done', (data) => {
         if (!data || !data.id) {
             return;
-        }
-
+        } 
+
```
Comment (coditva) [inline dashboard/frontend/actions/index.js:61]: We should add a lint rule to remove/error on extra white-spaces

Comment (coditva) [reply dashboard/frontend/actions/index.js:61]: Created ticket #40

File: dashboard/frontend/styles/_basics.scss [hunk 3]
```diff
@@ -4,7 +4,9 @@
     box-sizing: border-box;
   }
 
-  body {
-      width: 99%;
-  }
+body {
+    width: 100vw;
```
Comment (coditva) [inline dashboard/frontend/styles/_basics.scss:8]: Noice! 💯

**Review by coditva — APPROVED** [review]

**Pull request merged**